src/lib/controllers/ui-controller.js

```diff
@@ -1954,11 +1971,19 @@ export default class UIController {
       this.evc.registerListener(
         listenerName, selector, this.getSelectedText.bind(this), GenericEvt, customEv)
     }
+
+    this.evc.registerListener(listenerName + '-mousemove', selector, this.testMouseMove.bind(this), MouseMove)
   }
 
   registerAndActivateGetSelectedText (listenerName, selector) {
     this.registerGetSelectedText(listenerName, selector)
     this.evc.activateListener(listenerName)
+
+    this.evc.activateListener(listenerName + '-mousemove')
+  }
+
+  testMouseMove (event, domEvt) {
```

Comment:
Is this a "regular" production function? If so, it's probably better to rename it because "test" sounds confusing, one could think it is for testing (i.e. like unit testing) only.

Reply:
agree

Reply:
and see prior comment

Reply:
oh - sorry - it was only for developing process - removed